ISHPermissionKit/ISHPermissionsViewController.m

```diff
@@ -205,6 +205,10 @@ - (void)permissionRequestViewController:(ISHPermissionRequestViewController *)vc
         [self transitionToPermissionCategoryAtIndex:nextIndex];
         [self setCurrentIndex:self.currentIndex + 1];
     }
+
+    if (error != nil && self.errorBlock) {
+        self.errorBlock(vc.permissionCategory, (NSError * _Nonnull)error);
```

Comment:
It seems that when dismissing animated (line 200) the error block might be executed *before* the completion block. If `!delegate` that order would be reversed. Would this cause any issues on the caller's side?